packages/web/src/components/page/Page.tsx

```diff
@@ -157,58 +148,19 @@ export const Page = (props: PageProps) => {
     }
   }
 
-  const formattedTitle = title
-    ? `${title} ${messages.dotAudius}`
-    : messages.audius
-
   return (
     <>
-      {/* Title */}
-      <Helmet>
-        <title>{formattedTitle}</title>
-        <meta property='og:title' content={formattedTitle} />
-        <meta name='twitter:title' content={formattedTitle} />
-      </Helmet>
-
-      {/* Description */}
-      {description ? (
-        <Helmet encodeSpecialCharacters={false}>
-          <meta name='description' content={description} />
-        </Helmet>
-      ) : null}
-
-      {/* OG Description - This is the actual description of the content, for example a Track description */}
-      {ogDescription ? (
-        <Helmet encodeSpecialCharacters={false}>
-          <meta property='og:description' content={ogDescription} />
-          <meta name='twitter:description' content={ogDescription} />
-        </Helmet>
-      ) : null}
-
-      {/* Canonical URL */}
-      {canonicalUrl ? (
-        <Helmet encodeSpecialCharacters={false}>
-          <link rel='canonical' href={canonicalUrl} />
-          <meta property='og:url' content={canonicalUrl} />
-        </Helmet>
-      ) : null}
-
-      {/* Image */}
-      {image ? (
-        <Helmet encodeSpecialCharacters={false}>
-          <meta property='og:image' content={image} />
-          <meta name='twitter:image' content={image} />
-        </Helmet>
-      ) : null}
-
-      <meta property='og:type' content='website' />
-      <meta name='twitter:card' content='summary' />
-
-      {structuredData && (
-        <script type='application/ld+json'>
-          {JSON.stringify(structuredData)}
-        </script>
-      )}
+      <MetaTags
+        {...{
+          title,
+          description,
+          ogDescription,
+          image,
+          canonicalUrl,
+          structuredData,
+          noIndex
+        }}
```

Comment:
Can't decide if I like this or not 🤔
We often make a named object above or list them all out, but this is more concise

Reply:
I blame copilot 😂

I'll make a named object!